app/routes.js

```diff
@@ -1,7 +1,12 @@
 // const controller = require('./controllers/controller');
 const { healthCheck } = require('./controllers/healthCheck');
+const { albums, photos } = require('./controllers/albums');
```

Comment:
Add some declarativity to controller functions, like `getAlbums`, `getPhotos`, etc